packages/components/organisms/f-cookie-banner/src/components/LegacyBanner.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
+        <div :class="[$style['c-cookieWarning'], { [$style['c-cookieBanner--hidden']]: hideBanner }]">
+            <div :class="$style['c-cookieWarning-inner']">
+                <p>
+                    {{ $t('legacyBannerText') }}
+                    <a
+                        class="c-cookieWarning-link"
+                        :href="$t('cookiePolicyLinkUrl')">
+                        {{ $t('legacyBannerLinkText') }}
+                    </a>
+                </p>
+                <button
+                    class="c-cookieWarning-btn"
+                    data-test-id="cookieBanner-close-button"
+                    aria-label="Close"
+                    @click="$emit('hide-legacy-banner')" />
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    data () {
+        return {
+            hideBanner: false
+        };
+    }
+};
+</script>
+
+<style lang="scss" module>
+    .c-cookieWarning {
+        box-sizing: border-box;
+        background-color: $grey--darkest;
+        position: fixed;
+        bottom: 0;
+        width: 100%;
+        z-index: 99999990;
+    }
+
+    .c-cookieWarning-inner {
+        margin: 0 auto;
+        padding: spacing() spacing(x3) spacing() spacing();
+        overflow: hidden;
+
+            p {
+                @include font-size(caption);
+                color: $white;
+                text-align: center;
+                margin: 0 auto;
+
+                    a {
+                        color: $white;
+                    }
+            }
+
+            button {
+                text-size-adjust: 100%;
+                font-size: 100%;
+                line-height: 1.15;
+                margin: 0;
+                display: block;
+                position: absolute;
+                top: 12px;
+                right: 8px;
+                width: 10px;
+                height: 10px;
+                background: url('//dy3erx8o0a6nh.cloudfront.net/images/icon-close-banner.png') no-repeat 50%;
+                background-size: 10px 10px;
+                border: none;
+                cursor: pointer;
+            }
```

Comment:
Can we use our button component so we don't have to do this?

Reply:
@SachaZvetelman I'll chat with Maryia about adding the close-cross style to f-button - although I am worried that by the time we get that done, we'll need to delete the legacy banner code anyway (it's a stop-gap to keep feature parity for the UK, till the UK is ready for the new banner), but it'll then be there for someone else in the future

Reply:
Done